types/v56/types.go

```diff
@@ -1360,11 +1360,107 @@ type VM struct {
+	AdapterType         string     `xml:"AdapterType"`                   // The type of disk controller, e.g. IDE vs SCSI and if SCSI bus-logic vs LSI logic.
+	ThinProvisioned     *bool      `xml:"ThinProvisioned,omitempty"`     // Specifies whether the disk storage is pre-allocated or allocated on demand.
+	StorageProfile      *Reference `xml:"StorageProfile,omitempty"`      // Specifies reference to a storage profile to be associated with the disk.
+	OverrideVmDefault   bool       `xml:"overrideVmDefault"`             // Specifies that the disk storage profile overrides the VM's default storage profile.
+	Iops                *int64     `xml:"iops,omitempty"`                // Specifies the IOPS for the disk.
+	VirtualQuantity     *int64     `xml:"VirtualQuantity,omitempty"`     // The actual size of the disk.
+	VirtualQuantityUnit string     `xml:"VirtualQuantityUnit,omitempty"` // The units in which VirtualQuantity is measured.
+}
+
+// MediaSection from VM/VmSpecSection struct
+type MediaSection struct {
+	MediaSettings []*MediaSettings `xml:"MediaSettings"`
+}
+
+// MediaSettings from VM/VmSpecSection/MediaSection struct
+type MediaSettings struct {
+	DeviceId    string     `xml:"DeviceId,omitempty"`    // Describes the media device whose media mount is being specified here. This deviceId must match the RASD.InstanceID attribute in the VirtualHardwareSection of the vApp's OVF description.
+	MediaType   string     `xml:"MediaType,omitempty"`   // Specified the type of media that is mounted onto the device.
+	MediaState  string     `xml:"MediaState,omitempty"`  // Specifies the state of the media device.
+	MediaImage  *Reference `xml:"MediaImage,omitempty"`  // The media image that is mounted onto the device. This property can be 'null' which represents that no media is mounted on the device.
+	UnitNumber  int        `xml:"UnitNumber"`            // Specified the type of media that is mounted onto the device.
+	BusNumber   int        `xml:"BusNumber"`             //	The bus number of the media device controller.
+	AdapterType string     `xml:"AdapterType,omitempty"` // The type of controller, e.g. IDE vs SCSI and if SCSI bus-logic vs LSI logic
+}
+
+// CpuResourceMhz from VM/VmSpecSection struct
+type CpuResourceMhz struct {
+	Configured  int64  `xml:"Configured`             // The amount of resource configured on the virtual machine.
+	Reservation *int64 `xml:"Reservation,omitempty"` // The amount of reservation of this resource on the underlying virtualization infrastructure.
+	Limit       *int64 `xml:"Limit,omitempty"`       // The limit for how much of this resource can be consumed on the underlying virtualization infrastructure. This is only valid when the resource allocation is not unlimited.
+	SharesLevel string `xml:"SharesLevel,omitempty"` //	Pre-determined relative priorities according to which the non-reserved portion of this resource is made available to the virtualized workload.
```

Comment:
I don't understand this comment. Could it be rephrased differently?

Reply:
If you know how - please advice. It's from documentation ^^

Reply:
To suggest a change I'd have to understand it first